tools/check-versions-are-consistent.py

```diff
@@ -0,0 +1,35 @@
+import re
+import sys
+
+sys.path.insert(0, ".")
+
+from uaclient.version import __VERSION__
+
+
+def error(reason):
+    print(reason, file=sys.stderr)
+    sys.exit(1)
+
+
+def get_changelog_version():
+    with open("./debian/changelog", "r") as changelog_file:
```

Comment:
Ah you're suggesting a diff like this?
```diff
diff --git a/uaclient/version.py b/uaclient/version.py
index 93a7c750..1dfa9338 100644
--- a/uaclient/version.py
+++ b/uaclient/version.py
@@ -32,9 +32,5 @@ def get_version():
             out, _ = util.subp(cmd)
             return out.strip()
         except exceptions.ProcessExecutionError:
-            # Rely on debian/changelog because we are in a git-ubuntu or other
-            # packaging repo
-            cmd = ["dpkg-parsechangelog", "-S", "version"]
-            out, _ = util.subp(cmd)
-            return out.strip()
+            pass
     return __VERSION__
```

because after this PR, we will be sure there won't be a difference between the changelog version and the __VERSION__.

Yep that makes sense to me

Reply:
and sure we can use `dpkg-parsechanglog` here instead. I forgot about that tool